app/views/publishers/_instant_donation_modal.html.slim

```diff
@@ -5,6 +5,8 @@
         = image_tag "icn-donation-jar@1x.png", id: 'icn-donation-jar'
         h2.modal-title = t ".headline"
         p = t ".intro"
+        = hidden_field_tag 'preferred_currency', current_publisher.wallet.default_currency
```

Comment:
You should check to make sure the default currency is set.

Reply:
We may be okay here, it turns out if the user hasn't connected their wallet yet or selected a default currency then the value of `current_publisher.wallet.default_currency` is `USD`.

We need to have some default value for the rewards banner, so I think `USD` would be the most fitting

Reply:
Fixed